### PR TITLE
Add missing steps for Nova compute FFU

### PR DESCRIPTION
**Pull request by bogdando**

### CODE REVIEW

File: docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc [hunk 1]
```diff
@@ -68,6 +68,8 @@ spec:
 $ oc wait --for condition=Ready --timeout=300s Nova/nova
 ----
 
+. Complete xref:adopting-compute-services-to-the-data-plane_data-plane[Adopting Compute services to the {rhos_acro} data plane]
```
Comment (klgill) [inline docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc:71]:
```suggestion
. Complete the steps in xref:adopting-compute-services-to-the-data-plane_data-plane[Adopting Compute services to the {rhos_acro} data plane].
```
@bogdando Wouldn't customers have already completed "Adopting the Compute services to the RHOSO data plane? Should this step be a prerequisite instead?

**Review by klgill — CHANGES_REQUESTED** [review]

File: docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc (hunk 1 above)
Comment (bogdando) [reply docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc:71]: Technically, however, it is easy to miss that crucial step if surfing through the upgrade guide as a separate document. Let's keep it as a heads up?

Comment (bogdando) [reply docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc:71]: This is not a prerequisite, as it the configuration switch happens in the middle, not in the beginning of the Compute services adoption, and ends after EDPM adoption.

Comment (klgill) [reply docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc:71]: OK, sounds good.

**Review by klgill — APPROVED** [review]

**Review by jistr — APPROVED** [review]

**Pull request merged**